ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/OpenApiDcatDatasetController.java

```diff
@@ -12,7 +12,7 @@
 
 import static org.apache.jena.riot.WebContent.*;
 
-@Tag(name = "DCAT")
```

Comment:
these tags were done on purpose to group everything of DCAT together.. making tags individual defeats the purpose of a tag